networks/rpc/handler.go

```diff
@@ -327,14 +327,14 @@ func (h *handler) handleCallMsg(ctx *callProc, msg *jsonrpcMessage) *jsonrpcMess
 	switch {
 	case msg.isNotification():
 		h.handleCall(ctx, msg)
-		logger.Debug("Served "+msg.Method, "duration", time.Since(start))
+		logger.Trace("Served "+msg.Method, "duration", time.Since(start))
 		return nil
 	case msg.isCall():
 		resp := h.handleCall(ctx, msg)
 		if resp.Error != nil {
-			logger.Warn("Served "+msg.Method, "reqid", idForLog{msg.ID}, "duration", time.Since(start), "err", resp.Error.Message)
+			logger.Trace("Served "+msg.Method, "reqid", idForLog{msg.ID}, "duration", time.Since(start), "err", resp.Error.Message)
```

Comment:
Can we make it Debug? Sometimes RPCs do not return enough error contexts, in which case I resort to the node log.

Reply:
I am also concerned with this since debug log will not be caught by datadog or grafana. I hope some critical errors will be logged somewhere else.